server/notification-providers/matrix.js

```diff
@@ -0,0 +1,37 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const Crypto = require('crypto')
+
+class Matrix extends NotificationProvider {
+
+    name = "matrix";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully. ";
```

Comment:
space at the end is necessary?

Reply:
Copied verbatim from discord.js. Will remove in both.